src/build/build_step_test.go

```diff
@@ -507,6 +530,25 @@ func TestSha1SingleHash(t *testing.T) {
 	}
 }
 
+func newStateWithHashCheckers(label, hashFunction string, hashCheckers []string) (*core.BuildState, *core.BuildTarget) {
```

Comment:
this might be a good case for a `...string`?

Reply:
Yeah, I guess it a little nicer.